### DISCUSSION TITLE
108 ver

### DISCUSSION THREAD

**Discussion by funkymokey** [3kh0/ext-remover#616] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have a chromebook that randomly stopped updating on 108? Is there any exploit for this version?!?!!? I have lightspeed btw

**Comment by AshtonDavies** [comment, marked as answer]
Many have Lightspeed, and Chrome OS 108 has a lot to offer actually. Supported exploits are:
1. LoMoH (https://tiny.cc/LoMoH) - Crash the locked mode feature to disable extensions besides Hapara and Read&Write. (patched 111)
2. LTMEAT (https://tiny.cc/LTMEAT) - Freeze extensions by hanging them. (Patched 115, workaround: https://github.com/3kh0/ext-remover/discussions/671#discussioncomment-6105092)
3. Incognito DNS (158.101.114.159) - Captive portal window opens an incognito window. (Patched 114)
4. Sh1mmer (https://sh1mmer.me) - Unenroll Chromebook (patched 111, workaround: https://sh1mmer.me/#:~:text=How%20to%20use%20sh1mmer%20after%20the%20patch)
5. E-Halcyon (https://fog.gay) - Defog Chromebook by launching a temporarily unenrolled environment on a USB or SD card. (Use SH1MMER instead on 108)

**Reply by pazza2234** [reply]
if you shim, fakemurk and maybe murkmod. then you won't get caught and can do more things easaly.
